c_sharp/image_classification/model/augment_model.py

```diff
@@ -0,0 +1,20 @@
+import onnx
+import torch
+from onnxruntime_extensions import pnp
+
+mobilenet_model = onnx.load_model('mobilenetv2-12.onnx')
+augmented_model = pnp.SequentialProcessingModule(
+    pnp.PreMobileNet(224),
+    mobilenet_model,
+    pnp.PostMobileNet())
+
```

Comment:
Some comments in the code explaining what each line is doing, or link to doco on the ort-extensions repo, would help a user understand what is going on.